sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/application-offering-change-request.institutions.controller.ts

```diff
@@ -136,6 +136,8 @@ export class ApplicationOfferingChangeRequestInstitutionsController extends Base
     return {
       applicationNumber: application.applicationNumber,
       programId: application.currentAssessment.offering.educationProgram.id,
+      programIsActive:
+        application.currentAssessment.offering.educationProgram.isActive,
```

Comment:
Not a blocker for the PR. We must think about centralizing the `is active` logic (inside service or even a util) as we have upcoming tickets which is going to consider the effective end date as well. 

![image](https://github.com/bcgov/SIMS/assets/54600590/0c8a3c77-3f8e-4c42-a36a-0393947fac6c)